<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Center</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/pixmaps/45.gif</normaloff>:/new/prefix1/pixmaps/45.gif</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(80, 150, 205);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>378</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="havesharedLabel">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>You Have Shared:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0,0,0);</string>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Outline</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>item(s)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>378</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add files to share</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/pixmaps/playlist/add.png</normaloff>:/new/prefix1/pixmaps/playlist/add.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Remove the selected file</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/pixmaps/playlist/remove.png</normaloff>:/new/prefix1/pixmaps/playlist/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Update the shared list</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/pixmaps/update.png</normaloff>:/new/prefix1/pixmaps/update.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="renameButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>16</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Edit the name of item</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/pixmaps/rename.png</normaloff>:/new/prefix1/pixmaps/rename.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Clear the list (not share any file)</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/new/prefix1/pixmaps/clear.png</normaloff>:/new/prefix1/pixmaps/clear.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="selectedfileLabel">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;No item selected&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>228</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableWidget" name="fileList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 254, 254);
font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DropOnly</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::LinkAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="background">
        <color>
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="background">
        <color>
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
